clients/src/test/java/org/apache/kafka/clients/consumer/LeaderRackAwareCooperativeStickyAssignorTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.clients.consumer;
+
+import org.apache.kafka.clients.consumer.internals.AbstractStickyAssignor;
+
+public class LeaderRackAwareCooperativeStickyAssignorTest extends AbstractLeaderRackAwareAssignorTest {
+    @Override
+    public AbstractStickyAssignor getAssignor() {
+
+        return new LeaderRackAwareCooperativeStickyAssignor();
+    }
+}
```

Comment:
IMHO, we should avoid to use `extend` on test, we can use `@ParameterizedTest` to test different arguments or classes, WDYT?